types/three/three-core.d.ts

```diff
@@ -2072,11 +2084,19 @@ export class Loader {
     static Handlers: LoaderHandler;
 }
 
+/**
+* Interface for all loaders
+* CompressedTextureLoader don't extends Loader class, but have load method
+*/
+export interface AnyLoader {
+  load(url: string, onLoad?: (result: any) => void, onProgress?: (event: ProgressEvent) => void, onError?: (event: ErrorEvent) => void): any;
+}
+
 export interface LoaderHandler {
-    handlers: (RegExp | Loader)[];
+    handlers: (RegExp | AnyLoader)[];
 
-    add(regex: RegExp, loader: Loader): void;
-    get(file: string): Loader;
+    add(regex: RegExp, loader: AnyLoader): void;
+    get(file: string): AnyLoader;
```

Comment:
I noticed that the correct return type is `AnyLoader | null`. Would you mind improving it?